1.0/docs/devguide/properties.md

```diff
@@ -325,7 +325,7 @@ Example:
     });
 
 **Warning:** A single property observer shouldn't rely on any other properties,
-sub-properties, or paths because the observer can be called while these 
```

Comment:
Those are just extra spaces at the end of the line. I'm not sure if it's your text editor adding them, or just the fact that I'm mostly using Sublime Text nowadays, which removes the trailing spaces. (Or, mine is configured to remove trailing spaces.) Which editor are you using nowadays?